code/core/src/common/js-package-manager/JsPackageManager.ts

```diff
@@ -109,15 +109,15 @@ export abstract class JsPackageManager {
   abstract getModulePackageJSON(packageName: string): PackageJson | null;
 
   isStorybookInMonorepo() {
-    const turboJsonPath = findUpSync(`turbo.json`, { stopAt: getProjectRoot() });
-    const rushJsonPath = findUpSync(`rush.json`, { stopAt: getProjectRoot() });
-    const nxJsonPath = findUpSync(`nx.json`, { stopAt: getProjectRoot() });
+    const turboJsonPath = find.up(`turbo.json`, { last: getProjectRoot() });
+    const rushJsonPath = find.up(`rush.json`, { last: getProjectRoot() });
+    const nxJsonPath = find.up(`nx.json`, { last: getProjectRoot() });
 
     if (turboJsonPath || rushJsonPath || nxJsonPath) {
       return true;
     }
 
-    const packageJsonPaths = findUpMultipleSync(`package.json`, { stopAt: getProjectRoot() });
+    const packageJsonPaths = findFilesUp(['package.json']);
```

Comment:
is there a reason this one isn't a regular `find.up`?

it looks like the `findFilesUp` just exists to apply globs to each parent, but this isn't a glob

Reply:
It walks upwards returning all matches, so it returns more than one value